app/views/staff/index.html.erb

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
```

Comment:
You shouldn't need these extra html/head tags here. The layout file that you are using should include them